src/_gettsim/transfers/arbeitsl_geld_2/kindergelduebertrag.py

```diff
@@ -0,0 +1,111 @@
+"""Module for the calculation of the Kindergeldübertrag."""
```

Comment:
For some reason, using the correct Umlaut in the filename results in gibberish and I get a ruff error in the pre-commits.